src/main/java/core/basesyntax/dto/OperationType.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.dto;
+
+public enum OperationType {
+    BALANCE("b"),
+    SUPPLY("s"),
+    PURCHASE("p"),
+    RETURN("r");
+
+    OperationType(String symbol) {
+    }
+
+    public static OperationType getOperationType(String symbol) {
+        switch (symbol) {
+            case "b": {
+                return BALANCE;
+            }
+            case "s": {
+                return SUPPLY;
+            }
+            case "p": {
+                return PURCHASE;
+            }
+            case "r": {
+                return RETURN;
+            }
+            default: {
+                throw new IllegalArgumentException("Wrong operation type");
+            }
+        }
```

Comment:
here we can use `values()` method and Stream API

Reply:
changed